src/shared/extension/hooks/use-dao-handles.ts

```diff
@@ -0,0 +1,13 @@
+import { useCommandQuery } from 'shared/messaging';
+
+import { GetDaoHandlesCommand } from '../commands';
+
+export const useDaoHandles = (application: string) => {
+  return useCommandQuery({
+    staleTime: Number.POSITIVE_INFINITY,
```

Comment:
since default staleTime is now also infinity you can remove this1

Reply:
I don't know if it's a matter of some unmerged changes but without the staleTime it looks like the request are not cached.